Sources/Fuzzilli/Compiler/Compiler.swift

```diff
@@ -468,7 +468,14 @@ public class JavaScriptCompiler {
                 try enterNewScope {
                     let beginCatch = emit(BeginCatch())
                     if tryStatement.catch.hasParameter {
-                        map(tryStatement.catch.parameter.name, to: beginCatch.innerOutput)
+                        let parameter = tryStatement.catch.parameter
+                        switch parameter.parameter {
```

Comment:
Fixed with 6e8fde4f29a6aac7caa92e8f4674b069ff3f5956. Theoretically we could allow other parameter types than IdentifierParameter but practically this would mean making changes to the core of Fuzzilli for a case that will happens very rarely. Thus, I have changed this to a reduced version that only allows the case with IdentifierParameter